ImageItem/ImageItem.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * Agate styled ImageItem.
+ *
+ * @module agate/ImageItem
+ * @exports ImageItem
+ * @exports ImageItemBase
+ * @exports ImageItemDecorator
+ */
+
+import kind from '@enact/core/kind';
+import {ImageItem as UiImageItem} from '@enact/ui/ImageItem';
+import PropTypes from 'prop-types';
+import compose from 'ramda/src/compose';
+import React from 'react';
+import Spottable from '@enact/spotlight/Spottable';
+import {MarqueeDecorator, MarqueeController} from '@enact/ui/Marquee';
```

Comment:
### Our import rule

sorted by package name, case insensitive. Each group below separated by a blank line
External imports (e.g. react, react-dom, other enact packages)
Package imports (e.g ../Button, ../internal/deprecate)
Module imports (e.g. ./utils, ./InputSpotlightDecorator)
LESS import (e.g. ./Component.less)

```
import kind from '@enact/core/kind';
import {Subscription} from '@enact/core/internal/PubSub';
import {contextTypes} from '@enact/i18n/I18nDecorator';
import {isRtlText} from '@enact/i18n/util';
import Changeable from '@enact/ui/Changeable';
import Pure from '@enact/ui/internal/Pure';
import PropTypes from 'prop-types';
import React from 'react';
 
import $L from '../internal/$L';
import Skinnable from '../Skinnable';
import Tooltip from '../TooltipDecorator/Tooltip';
 
import InputDecoratorIcon from './InputDecoratorIcon';
import InputSpotlightDecorator from './InputSpotlightDecorator';
import {calcAriaLabel, extractInputProps} from './util';
 
import componentCss from './Input.less';
```

Reply:
Thanks for clarification 👍